dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -55,12 +54,12 @@ interface Props {
 export const StickyVideo = ({ isPlaying, children }: Props) => {
 	const [isSticky, setIsSticky] = useState(false);
 	const [isIntersecting, setRef] = useIsInView({
-		threshold: 0.1,
+		threshold: 0.5,
 		repeat: true,
 	});
 
 	useEffect(() => {
-		setIsSticky(isIntersecting && isPlaying);
+		if (isPlaying) setIsSticky(!isIntersecting);
```

Comment:
This might cause a bug if isPlaying changes to false but we still want to unstick a stuck element

Reply:
Ah yes, hadn't thought of that..